src/HasProfilePhoto.php

```diff
@@ -66,7 +67,11 @@ public function getProfilePhotoUrlAttribute()
      */
     protected function defaultProfilePhotoUrl()
     {
-        return 'https://ui-avatars.com/api/?name='.urlencode($this->name).'&color=7F9CF5&background=EBF4FF';
+        $name = trim(collect(explode(' ', $this->name))->map(function ($segment) {
+            return $segment[0] ?? '';
```

Comment:
@taylorotwell, @geisi as mentioned in a comment at https://github.com/laravel/framework/pull/40381, this implementation doesn't work as expected with multi-byte letters (eg. `Á`).

It should be either changed to `return Str::substr($segment, 0, 1);`, or the linked PR should be included in the framework package and its new function should be used here like this:

```
return 'https://ui-avatars.com/api/?name='.urlencode(Str::initials($this->name)).'&color=7F9CF5&background=EBF4FF';
```